InjectionTDD/Classes/TDDTestKeeper.m

```diff
@@ -22,13 +22,56 @@ +(void)initialize{
     
     TDDInjectionTestsObserver *observer = [TDDInjectionTestsObserver new];
     [XCTestObservationCenter.sharedTestObservationCenter addTestObserver:observer];
-    
-    if (![[NSBundle bundleWithPath: @"/tmp/injectionforxcode/InjectionLoader.bundle"] load]) {
-        NSLog (@"Warning: Injection for Xcode not found. Please open Injection For Xcode.");
-    }else{
+
+    NSString *injectionForXcode = @"/tmp/injectionforxcode";
+    NSString *injectionIII = @"/Applications/InjectionIII.app/Contents/Resources/";
+
+
+#ifdef TARGET_OS_IOS
+    NSString *platformString = @"iOSInjection{version}.bundle";
+#endif
+
+#ifdef TARGET_OS_TV
+    platformString = @"tvOSInjection{version}.bundle";
+#endif
+
+#ifdef TARGET_OS_MACOS
+    NSString *platformString = @"macOSInjection{version}.bundle";
+#endif
+
+    NSArray *paths = [NSArray arrayWithObjects:
+                      injectionForXcode,
+                      injectionIII,
```

Comment:
From you comment you mention that InjectionIII has a precedence (I totally agree) but here you place `injectionIII` before `injectionForXcode`. Is it intentional?

Reply:
Oh, that was a mistake on my part, I'll fix that right up!